.github/workflows/linux.yml

```diff
@@ -59,22 +62,27 @@ jobs:
           QT_PKGS: ${{ matrix.qt-version-major == '6' && 'qt6-base-private-dev libgl-dev' || 'qtbase5-private-dev' }}
         run: |
           sudo apt update
-          sudo apt install -y build-essential qtchooser ${{ env.QT_PKGS }}
+          sudo apt install -y build-essential qtchooser ${{ env.QT_PKGS }} cmake ninja
```

Comment:
Please remove `qtchooser`, it was only necessary for `qmake`. And please install the packages in alphabetical order? Thanks 🥺❤️